### DISCUSSION TITLE
Cant access admin on digitalocean deployment

### DISCUSSION THREAD

**Discussion by wetterlicht** [strapi/strapi#8947] [Legacy Topics]
Hi,
I've been following the tutorial for deploying on digitalocean and everything seemed to work fine. The server is running successfully under <myip>:1337 in production mode. 
However when I try to access <myip>:1337/admin the page gets stuck on the loading animation.  The console reveals `Failed to load resource: net::ERR_BLOCKED_BY_CLIENT localhost:1337/admin/init:1 `. I'm not sure if I've misconfigured something or why the browser is trying to access localhost instead of <myip>. Would appreciate any pointers.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
